src/app/payment/paymentMethod/StripePaymentMethod.tsx

```diff
@@ -1,9 +1,9 @@
-import { PaymentInitializeOptions, StripeElementOptions } from '@bigcommerce/checkout-sdk';
+import { CardInstrument, PaymentInitializeOptions, StripeElementOptions } from '@bigcommerce/checkout-sdk';
 import React, { useCallback, FunctionComponent } from 'react';
-import { Omit } from 'utility-types';
```

Comment:
Hi, could we confirm `Omit` can be removed safely? It is still used on line 11.

Reply:
Hi @bc-peng, apparently yes. There're no errors on circle ci.

Reply:
Yes, it is not necessary to import this since `TS 3.5`.
https://www.typescriptlang.org/docs/handbook/utility-types.html